core/src/main/java/nl/nn/adapterframework/util/JdbcUtil.java

```diff
@@ -776,6 +776,8 @@ public static SQLType mapParameterTypeToSqlType(ParameterType parameterType) {
 				return JDBCType.CLOB;
 			case BINARY:
 				return JDBCType.BLOB;
+			case LIST:
+				return dbmsSupport.getCursorSqlType();
```

Comment:
Misschien wel handig hier bij te zetten dat dit nu alleen gedaan is voor de StoredProcedures, en dat we zodra er meedere soorten Parameters hebben we dit een 'correct' naampje gaan geven.